src/sbvr-api/user.ts

```diff
@@ -0,0 +1,214 @@
+import type { Types } from '@balena/abstract-sql-to-typescript';
```

Comment:
We could do similar as we do on balena-api to generate a line on top saying this file is auto-generated, it can avoid people mistakenly modifying stuff here

Reply:
That makes a lot of sense, so much so that I think `@balena/abstract-sql-to-typescript` should do it so it's universal (and include the version used to generate the types)

Reply:
https://github.com/balena-io-modules/abstract-sql-to-typescript/pull/68